.lintstagedrc

```diff
@@ -0,0 +1,4 @@
+{
+  "*.{js,css,md,html,json}": "prettier --write",
+  "*.{js, jsx}": "eslint --fix ."
```

Comment:
as I remember, --fix flag will try to automatically fix all the mistakes he can in your code, usually it's not what we want, because more errors might be introduced to the code
please, look it up in the documentation

Reply:
I thought it would be better to immediately fix the errors that eslint can fix, other errors will show up. But if it is necessary to change, then I changed.

Reply:
linter configured to fix errors it can, but that approach is error prone, because your code may be created with intention to work that way